src/main/java/techcourse/myblog/application/ArticleService.java

```diff
@@ -1,89 +1,90 @@
-        User author = userRepository.findById(userResponse.getId())
-                .orElseThrow(() -> new NoUserException("유저가 존재하지 않습니다."));
+    public Long save(ArticleDto articleDto, UserResponse userResponse) {
+        User author = userService.findById(userResponse.getId());
 
-        Article article = modelMapper.map(articleDto, Article.class);
-        article.setAuthor(author);
+        Article article = articleAssembler.convertToArticle(articleDto, author);
         Article savedArticle = articleRepository.save(article);
         return savedArticle.getId();
     }
 
-    public Article findArticleById(Long articleId) {
+    public ArticleDto find(Long articleId, UserResponse userResponse) {
+        return articleAssembler.convertToDto(findByUser(articleId, userResponse));
+    }
+
+    public ArticleDto find(Long articleId) {
+        return articleAssembler.convertToDto(findById(articleId));
+    }
+
+    public List<ArticleDto> findAll() {
+        List<ArticleDto> articles = articleRepository.findAll().stream()
+                .map(articleAssembler::convertToDto)
+                .collect(Collectors.toList());
+        return Collections.unmodifiableList(articles);
+    }
+
+    public Article findById(Long articleId) {
         return articleRepository.findById(articleId)
                 .orElseThrow(() -> new NoArticleException("해당 게시물은 존재하지 않습니다!"));
     }
 
-    public Article findArticleWrittenByUser(Long articleId, UserResponse userResponse) {
-        Article article = findArticleById(articleId);
-        User user = modelMapper.map(userResponse, User.class);
+    private Article findByUser(Long articleId, UserResponse userResponse) {
+        Article article = findById(articleId);
+        User author = userService.findById(userResponse.getId());
 
-        if (!article.isSameAuthor(user)) {
+        if (!article.isSameAuthor(author)) {
             throw new NotSameAuthorException("해당 작성자만 글을 수정할 수 있습니다.");
         }
 
         return article;
     }
 
     @Transactional
-    public void editArticle(ArticleDto articleDto, Long articleId, UserResponse userResponse) {
-        Article article = articleRepository.findById(articleId)
-                .orElseThrow(() -> new NoArticleException("해당 게시물은 존재하지 않습니다!"));
-
-        User author = userRepository.findById(userResponse.getId())
-                .orElseThrow(() -> new NoUserException("유저가 존재하지 않습니다."));
+    public ArticleDto modify(ArticleDto articleDto, Long articleId, UserResponse userResponse) {
+        Article article = findByUser(articleId, userResponse);
+        User author = userService.findById(userResponse.getId());
 
-        if (!article.isSameAuthor(author)) {
-            throw new NotSameAuthorException("해당 작성자만 글을 수정할 수 있습니다.");
-        }
-
-        article.updateArticle(modelMapper.map(articleDto, Article.class));
+        article.updateArticle(articleAssembler.convertToArticle(articleDto, author));
```

Comment:
```suggestion
        article.update(articleAssembler.convertToArticle(articleDto, author));
```
그리고 convert의 결과가 무엇인지 지역변수로 두는 것도 의미가 있다고 생각합니다.